integration/graphql-types.ts

```diff
@@ -444,6 +466,52 @@ function enumOrDetailOrNullOfPopularity(
     });
   }
 }
+const enumDetailNameOfPopularity = {
+  Low: "Low",
+  High: "High",
+};
+
+function enumOrDetailOfPopularity(
+  enumOrDetail: PopularityWithExtraFieldDetailOptions | Popularity | undefined,
+): PopularityWithExtraFieldDetail {
+  if (enumOrDetail === undefined) {
+    return newPopularityWithExtraFieldDetail();
+  } else if (typeof enumOrDetail === "object" && "code" in enumOrDetail) {
+    return {
+      __typename: "PopularityWithExtraFieldDetail",
+      code: enumOrDetail.code!,
+      name: enumDetailNameOfPopularity[enumOrDetail.code!],
+      ...enumOrDetail,
+    };
+  } else {
+    return newPopularityWithExtraFieldDetail({
+      code: enumOrDetail as Popularity,
+      name: enumDetailNameOfPopularity[enumOrDetail as Popularity],
+    });
+  }
+}
+
+function enumOrDetailOrNullOfPopularity(
+  enumOrDetail: PopularityWithExtraFieldDetailOptions | Popularity | undefined | null,
+): PopularityWithExtraFieldDetail | null {
```

Comment:
Huh... I think something strange happened since I used the `Popularity` enum twice...

Reply:
Changed to use the `Working` enum instead 👍